ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -637,14 +637,15 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)(
       pageMode
     );
 
-    const showCtRenewalBanner =
+    const ctNeedsRenewal =
       pageMode === DARPageMode.ANNUAL_RENEWAL &&
       isCompliant(getAccessModuleStatusByName(profile, ctModule)) &&
       !isRenewalCompleteForModule(
         getAccessModuleStatusByName(profile, ctModule)
       );
-    const showCompletionBanner =
-      profile && !nextRequired && !showCtRenewalBanner;
+    const showCtRenewalBanner =
+      enableControlledTierTrainingRenewal && ctNeedsRenewal;
+    const showCompletionBanner = profile && !nextRequired && !ctNeedsRenewal;
```

Comment:
It's still misleading to show the completion banner when the user needs CT renewal, regardless of whether it's possible to do it.